src/renderer/classes/outputManager/MemberChannel.js

```diff
@@ -1,106 +1,97 @@
 import '../../typedef';
 import { NOTE_ON, MODULATION, NOTE_OFF } from './note-actions';
-import { deriveStatusByte } from '../../utils/midi/statusByteUtils';
-import * as midiMessageType from '../../utils/midi/midi-message-types';
 import * as defaults from '../../constants/defaults';
+import {
+  noteOffMessage, noteOnMessage, pitchBendMessage, channelPressureMessage, cc74Message
+} from '../../utils/midi/formatMidiMessage';
+import { PITCH_BEND, PRESSURE, TIMBRE } from './modulation-types';
+import { pick } from 'lodash';
 
 class MemberChannel {
   /**
-   * @param  {number} midiChannel integer from 1 to 16
-   * @param  {function} nowFn function to current time. e.g. performance.now
+   * @param  {object} param
+   * @param  {number} param.midiChannel Integer from 1 to 16
+   * @param  {function} param.nowFn Function to get current time. e.g.`performance.now`
    */
-  constructor (midiChannel, nowFn) {
+  constructor ({ midiChannel, nowFn }) {
     this.nowFn = nowFn;
     this.timeOfLastNoteOff = nowFn();
     this.activeNoteOn = null;
     this.midiChannel = midiChannel;
+    this.pitchBendRange = 48;
   };
   /**
-   * @param {string} actionType
-   * @param  {NoteOn | Modulation | NoteOff } [action]
+   * @param  {NoteOn | Modulation | NoteOff } noteAction
+   * @param  {object} [options]
+   * @param  {number} [options.pitchBendRange] 0 to 96.
    */
-  deriveMidiMessage (actionType, action) {
-    this.messages = [];
-    switch (actionType) {
+  deriveMidiMessage (noteAction, options = {}) {
+    if (options.pitchBendRange != null) this.pitchBendRange = options.pitchBendRange;
```

Comment:
This negation of a falsy value actually expects a truthy value, right?
```javascript
if(options.pitchBendRange){
  this.pitchBendRange = options.pitchBendRange;
}
```

is much easier to read.

Reply:
Alternatively, you could write 
```javascript
this.pitchBendRange = options.pitchBendRange || this.pitchBendRange;
```
but I wouldn't find this sophisticated in this case.

Reply:
`options.pitchBendRange` is a number and can be `0`. And `0` should return true.